README.md

```diff
@@ -3,176 +3,28 @@ layout: page
 title: User Guide
 ---
 
-AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
+[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
```

Comment:
Should this be linked to our repo's actions instead of the se-edu's one?

Reply:
Ya but i dont know what badges we should add, there's quite a few different one

Reply:
no i mean the link not the badges. Now you are linking it to se-edu/addressbook-level3

Reply:
https://github.com/AY2021S1-CS2103-T14-3/tp/actions
this

Reply:
Changed